src/network/networkd-route.c

```diff
@@ -1107,10 +1107,10 @@ int config_parse_tcp_window(const char *unit,
         if (r < 0)
                 return r;
 
-        r = safe_atou32(rvalue, &k);
-        if (r < 0) {
+        r = parse_size(rvalue, 1024, &k);
```

Comment:
Now `k` may be larger than `UINT32_MAX`. I think we should check the value is not larger than `UINT32_MAX`.